tests/unittest_settings.py

```diff
@@ -859,4 +859,8 @@
 
 
 DISABLED_MODELS = ("EntityDividerModel",)
-EMAIL_HOST = ''
+
```

Comment:
This file should not be changed by this PR

Reply:
removed